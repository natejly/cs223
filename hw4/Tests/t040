#!/bin/bash
# 4 Battlefields, 3 Entries, Different Scoring

trap "/usr/bin/killall -q -u $USER ./Blotto 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./Blotto ]; then
  echo './Blotto is missing or not executable'
  echo './Blotto is missing or not executable' 1>&2
  exit 1
fi

$BIN/run -stderr=/dev/null ./Blotto 1 3 4 6 <$WHERE/t040.in
