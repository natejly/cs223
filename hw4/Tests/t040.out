P1 8.0 - P2 6.0
P1 8.0 - P3 6.0
P2 10.0 - P3 4.0
